.github/workflows/check-go-task.yml

```
@@ -25,8 +26,36 @@ on:
   repository_dispatch:
 
 jobs:
+  run-determination:
+    runs-on: ubuntu-latest
+    outputs:
+      result: ${{ steps.determination.outputs.result }}
+    steps:
+      - name: Determine if the rest of the workflow should run
+        id: determination
+        run: |
+          RELEASE_BRANCH_REGEX="refs/heads/[0-9]+.[0-9]+.x"
+          # The `create` event trigger doesn't support `branches` filters, so it's necessary to use Bash instead.
+          if [[ \
+            "${{ github.event_name }}" != "create" || \
+            ( \
+              "${{ github.event_name }}" == "create" && \
+              "${{ github.ref }}" =~ $RELEASE_BRANCH_REGEX \
+            ) \
```

This could be simplified to:

```suggestion
            "${{ github.event_name }}" != "create" || \
            "${{ github.ref }}" =~ $RELEASE_BRANCH_REGEX \
```

---

Thanks Cristian! I make the suggested change in https://github.com/arduino/arduino-cli/compare/fa7d20febfaa279c92f645e23a81f2df8d8e4e01..73146cef2b8b62732726efb898ad9c8235cc704d